src/meta/src/backup_restore/metrics.rs

```diff
@@ -12,19 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::ops::Deref;
+use std::sync::LazyLock;
+
 use prometheus::{
     exponential_buckets, histogram_opts, register_histogram_vec_with_registry,
-    register_int_counter_with_registry, Histogram, IntCounter, Registry,
+    register_int_counter_with_registry, Histogram, IntCounter,
 };
+use risingwave_common::monitor::GLOBAL_METRICS_REGISTRY;
+
+pub static GLOBAL_BACKUP_MANAGER_METRICS: LazyLock<BackupManagerMetrics> =
+    LazyLock::new(BackupManagerMetrics::new);
 
+#[derive(Clone)]
 pub struct BackupManagerMetrics {
     pub job_count: IntCounter,
     pub job_latency_success: Histogram,
     pub job_latency_failure: Histogram,
 }
 
+impl Default for BackupManagerMetrics {
+    fn default() -> Self {
+        GLOBAL_BACKUP_MANAGER_METRICS.clone()
+    }
+}
+
 impl BackupManagerMetrics {
-    pub fn new(registry: Registry) -> Self {
+    fn new() -> Self {
```

Comment:
Do we really need these `XxxxxxMetrics` anymore? From my understanding, they are basically an alias of `GLOBAL_METRICS_REGISTRY` now.

By the way, now it's also a little bit weird to register Metrics in a `new()` function.

Reply:
Considering `new` is now private and only used to initialize the `GLOBAL_XXX_METRICS`, I guess it's okay. However, I guess it might be more clear to still take `registry` as the argument, while initializing `GLOBAL` with `new(GLOBAL_REGISTRY)`.

Reply:
> Do we really need these XxxxxxMetrics anymore? From my understanding, they are basically an alias of GLOBAL_METRICS_REGISTRY now.

I think we still need them. They are not an alias to GLOBAL_METRICS_REGISTRY. Each `XxxxxMetrics` has its own global instance, named `GLOBAL_XXXXX_METRICS`. Each `GLOBAL_XXXX_METRICS` registers to `GLOBAL_METRICS_REGISTRY` when they get initialized. 

Maintaining different `XxxxxMetrics` makes our metrics clearer, since they are maintained in different struct related to the modules they report. And more importantly, the `GLOBAL_XXXX_METRICS` gets initialized when they are used the first time. Therefore, we can avoid registering some metrics not used in a node, such as storage metrics on frontend node. If we maintain all the metrics in a global struct, all metrics will be registered in all nodes.

Reply:
> However, I guess it might be more clear to still take registry as the argument, while initializing GLOBAL with new(GLOBAL_REGISTRY).

I have changed it.

Reply:
I see. `GLOBAL_XXXX_METRICS` manages the different metrics lifecycle, including registering. Thanks for explanation!